src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,139 @@
+package duke;
+
+/**
+ * Ui class provides methods for user interaction
+ */
+public class Ui {
+    private final String line = "_______________________________________________________\n";
+
+    /**
+     * Shows a welcome text
+     * @return a String representing welcome text
+     */
+    public String showWelcome() {
+        return line + " Hello! I'm Duke\n" + " What can I do for you?\n" + line;
+    }
```

Comment:
There are many instances of `line + "something" + line`. Perhaps, you could consider having a private method that does that?